text/0068-feat-login-into-portable-account-with-metamask.md

```diff
@@ -0,0 +1,116 @@
+- FIP: 68
+- title: Login into Portable account with metamask 
+- author: Sabyasachi Patra, @asabya
+- status: draft
+- created: 2023-03-01
+
+# Summary
+Addressing web3 login workflow to portable account where users can retrieve their wallet in a decentralized manner from browser with metamask.

```

Comment:
the implementation details do not elaborate how the proposed architecture could be implemented with Metamask it only has go code examples.

Reply:
It will depend on the dApp how it implements the frontend logic. here is a [demo](https://github.com/fairDataSociety/fairos-client-examples/blob/a4c16556c6b10a2d2b3780a74a9978f090ab31e8/wasm-hello-world/src/App.tsx#L8) implementation